### PR TITLE
Add singleLineAllowed option to ForBraceChecker

### LINKED ISSUE #286: Add singleLineAllowed option to ForBraceChecker
Similar to the option of the same name in `IfBraceChecker`

### PR DESCRIPTION
**Pull request by marconilanna**
Fixes #286

### CODE REVIEW

File: src/main/scala/org/scalastyle/scalariform/ForBraceChecker.scala [hunk 1]
```diff
@@ -16,21 +16,36 @@
...
+      )
     } yield PositionError(t.lParenOrBrace.offset)
   }
+
+  private def validSingleLine(t: ForExpr, lines: Lines) = {
+    singleLineAllowed && {
+      val singleLine = for {
+        start <- lines.toLineColumn(t.forToken.offset)
+        end <- lines.toLineColumn(t.tokens.last.offset)
+        if start.line == end.line
```
Comment (marconilanna) [inline src/main/scala/org/scalastyle/scalariform/ForBraceChecker.scala]: @matthewfarwell please let me know whether this is the best way to implement it

**Comment by marconilanna** [comment]
@matthewfarwell bump

**Comment by matthewfarwell** [comment]
Thanks for the bump. I'm really busy at the minute, but I'll get to this next week.

**Comment by matthewfarwell** [comment]
Great. Thanks!

**Pull request merged**